contributors/design-proposals/architecture/scope.md

```diff
@@ -0,0 +1,342 @@
+* [Kubernetes architectural roadmap (2017)](architectural-roadmap.md)
+* [Design principles](principles.md)
+* [Kubernetes resource management](resource-management.md)
+
+Kubernetes is a portable, extensible open-source platform for managing
+containerized workloads and services, that facilitates both
+declarative configuration and automation.  Workload portability is an
+especially high priority. Kubernetes provides a flexible, easy-to-run,
+secure foundation for running containerized applications on any cloud
+provider or your own systems.
+
+While not a full distribution in the Linux sense, adoption of
+Kubernetes has been facilitated by the fact that the upstream releases
+are usable on their own, with minimal dependencies (e.g., etcd, a
+container runtime, and a networking implementation). 
+
+The high-level scope and goals are often insufficient for making
+decisions about where to draw the line, so this documents where the
+line is, the rationale for some past decisions, and some general
+criteria that have been applied, including non-technical
+considerations. For instance, user adoption and continued operation of
+the project itself are also important factors.
+
+## Significant areas
+
+More details can be found below, but a concise list of areas in scope follows:
+* Containerized workload execution and management
+* Service discovery, load balancing, and routing
+* Workload identity propagation and authentication
+* Declarative resource management platform
+* Command-line tool
+* Web dashboard (UI)
+* Cluster lifecycle tools
+* Extensibility to support execution and management in diverse environments
+* Multi-cluster management tools and systems
+* Project GitHub automation and other process automation
+* Project continuous build and test infrastructure
+* Release tooling
+* Documentation
+* Usage data collection mechanisms
+
+## Scope domains
+
+Most decisions are regarding whether any part of the project should
+undertake efforts in a particular area. However, some decisions may
+sometimes be necessary for smaller scopes. The term "core" is sometimes
+used, but is not well defined. The following are scopes that may be relevant:
+* Kubernetes project github orgs
+  * All github orgs
+  * The kubernetes github org
+  * The kubernetes-sigs and kubernetes-incubator github orgs
+  * The kubernetes-client github org
+  * Other github orgs
+* Release artifacts
+  * The Kubernetes release bundle
+  * Binaries built in kubernetes/kubernetes
+    * “core” server components: apiserver, controller manager, scheduler, kube-proxy, kubelet
+    * kubectl
+    * kubeadm
+  * Other images, packages, etc.
+* The kubernetes/kubernetes repository (aka k/k)
+  * master branch
+  * kubernetes/kubernetes/master/pkg
+  * kubernetes/kubernetes/master/staging
+* [Functionality layers](architectural-roadmap.md)
+  * required
+  * pluggable
+  * optional
+  * usable independently of the rest of Kubernetes
+
+## Other inclusion considerations
```

Comment:
Is the message intended to be that the answer to *all* of these questions needs to be 'yes' in order to be considered in scope?  Or mostly yes?  Or some combination? It seems that for a minority of the questions, an answer of 'no' is required, or at least an answer of 'yes' motivates for exclusion.  For example  "Is there an acceptable home for the recommended ecosystem solution(s)?"

Reply:
In summary, needs some explanation or rewording to clarify.

Reply:
Ack